source/includes/usage-examples/write-code-examples.c

```diff
@@ -0,0 +1,142 @@
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int
+main (int argc, char *argv[])
+{
+    mongoc_client_t *client;
+    mongoc_collection_t *collection;
+    char *str;
+    bson_error_t error;
+
+    mongoc_init ();
+
+    client =
+        mongoc_client_new ("<connection string URI>");
+    collection =
+        mongoc_client_get_collection (client, "<database name>", "collection name");
+
+{
+    // start-insert-one
+    bson_t *document = BCON_NEW ("<field name>", BCON_UTF8 ("<value>"));
+    bson_error_t error;
+
+    if (!mongoc_collection_insert_one (
+        collection, document, NULL, NULL, &error)) {
+        fprintf (stderr, "Insert one operation failed: %s\n", error.message);
+    }
+    bson_destroy (document);
+    //end-insert-one
+}
+
+{
+    // start-insert-many
+    size_t num_docs = 2;
+    bson_t *docs[num_docs];
+    docs[0] = BCON_NEW ("<field name>", BCON_UTF8 ("<value>"));
+    docs[1] = BCON_NEW ("<field name>", BCON_UTF8 ("<value>"));
+
+        bson_error_t error;
```

Comment:
S: Seems like there are some extra spaces here
```suggestion
    bson_error_t error;
```

Reply:
Applies to all the lines in this section after this one